### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href$="forum\.openage\.dev"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [SFTtech/openage#1449]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[openage - free, modern RTS game engine](https://openage.sft.mx/)
#### Element path
a[href$="forum\.openage\.dev"]
#### Snippet
`<a href="https://forum.openage.dev" id="button">Forum</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 1.37 (foreground color: #dcdcdc, background color: #ffffff, font size: 10.3pt (13.68px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
